piet-wgsl/src/simple_text.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 The piet-gpu authors.
```

Comment:
Is this here just because the demo hasn't been ported over to using parley? I can't imagine we'll want to maintain this long term.

Reply:
Exactly. My goal was to bring the wgsl crate up to parity with piet-gpu so we can drop the old code.

I intend to submit another PR to peniko soon that will add a collection of font types (specifically the resource type itself, but also the styling properties) after which I'll update parley to depend on that and we'll have much better cohesiveness among these crates.